Cargo.toml

```diff
@@ -59,6 +59,7 @@ thread_profiler = { version = "0.1", optional = true }
 amethyst_gltf = { path = "amethyst_gltf", version = "0.1" }
 genmesh = "0.5"
 amethyst_animation = { path = "amethyst_animation", version = "0.1.0" }
+ron = "0.1"
```

Comment:
ron version is 0.2, any reason to use 0.1?

Reply:
To keep it consistent with `amethyst_assets` and `amethyst_config`. Generally within a repo I try to keep the same dependency versions

Reply:
okay, then we might want to update it in a future PR. For this one you can keep 0.1